amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/job/DispositionLifecycleJobExecuter.java

```diff
@@ -166,56 +202,59 @@ protected String getQuery()
     /**
      * @see org.alfresco.module.org_alfresco_module_rm.job.RecordsManagementJobExecuter#execute()
      */
+    @Override
     public void executeImpl()
     {
         try
         {
             logger.debug("Job Starting");
 
-            if (dispositionActions != null && !dispositionActions.isEmpty())
+            if (dispositionActions == null || dispositionActions.isEmpty())
```

Comment:
if I understand right  if the job starts and this is empty ...in the logs, we don't see that the Job also finished...and this doesn't make sense for me

Reply:
I have added a Log entry.